README.md

```diff
@@ -238,11 +238,34 @@ Simplified Login comes with the following localization support:
 1. 🇩🇰 Danish
 1. 🇬🇧 English (Default)
 
-#### Releasing
+## Local development
 
-1. Make sure all changes going in the release have been merged to `master` branch.
+### Setup
+
+## Prerequisites
+1. All you need is Xcode with command-line tools installed. It works perfectly with version 13.4.1.
+
+## Steps
+1. Clone repository
+1. Open workspace.xcworkspace file, which exists in the Projects folder. It contains two projects inside: AccountSDKIOSWeb which is the SDK itself, and ExampleWeb which serves as a demo application.
```

Comment:
Formatting (nit): Would be nice to stylize the filename `workspace.xcworkspace`.